test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -53,6 +53,9 @@ export default class PerfCompare extends Component {
         	inputURL[url] = urlData[url];
         }
         await this.setState({inputURL: inputURL});
+        if(this.state.inputURL){
+            this.handleParseInputURL();
+        }
```

Comment:
- We do not need to `await` for setState(). However, `handleParseInputURL ` needs `await`
- Why we need `if(this.state.inputURL)`? can inputURL be empty? If it can, why we set it into state?

Reply:
- agree
- inputURL be empty when we first click the perfCompare button. This  inputURL not empty case comes from tabularView(click the Trafficlight cell would directly come to PerfCompare page)

Reply:
Basically, this check saved the submit button from been clicked in perfCompare from TabularView. As described from the https://github.com/AdoptOpenJDK/openjdk-test-tools/issues/227